pkg/lang/ir/v1/system.go

```diff
@@ -46,6 +47,85 @@ func (g generalGraph) compileUbuntuAPT(root llb.State) llb.State {
 	return root
 }
 
+func (g generalGraph) copyAptSignature(root llb.State, sign string, url string) llb.State {
+	base := llb.Image(builderImage)
+	builder := base.
+		File(llb.Mkdir("/etc/apt/keyrings", 0755, llb.WithParents(true)),
+			llb.WithCustomName("[internal] setting bulder apt-source signature folder")).
+		File(llb.Mkfile(sign, 0644, []byte("")),
+			llb.WithCustomName("[internal] creating apt-source signature file")).
+		Run(llb.Shlex(fmt.Sprintf("sh -c \"%s\"", "curl "+url+" >> "+sign)),
+			llb.WithCustomName("[internal] downloading apt-source signature in base image")).Root()
```

Comment:
This can be placed in any file. You don't need to create it on the same path.

BTW, `fmt.Sprintf` appears with `+` is weird.

Reply:
Directly download the signature to the default path